gen/scim_atlassian_mu

```diff
@@ -0,0 +1,134 @@
+#!/usr/bin/perl
+
+use strict;
+use warnings;
+use perunServicesInit;
+use perunServicesUtils;
+use open qw/:std :utf8/;
+use JSON::XS;
+use utf8;
+
+local $::SERVICE_NAME = "scim_atlassian_mu";
```

Comment:
SCIM is the protocol here so I would prefer to not be part of the service's name. So maybe "atlassian_mu" would be better, what do you think?